src/bundle/Resources/views/themes/admin/account/profile/view.html.twig

```diff
@@ -58,22 +58,25 @@
                     {% set items = [
                         {
                             label: 'user_profile.summary.full_name.label'|trans|desc("Full name"),
-                            content: ibexa_content_name(content)
+                            content: ibexa_content_name(content),
                         },
                         {
                             label: 'user_profile.summary.email.label'|trans|desc("Email"),
-                            content_raw: email_link
+                            content_raw: email_link,
                         }
                     ] %}
 
                     {% for group in field_groups %}
                         {% set field_definitions = content.getContentType().getFieldDefinitions().filterByGroup(group) %}
 
                         {% for field_definition in field_definitions %}
+                            {% set undefined_label = 'user_profile.summary.undefined'|trans|desc('undefined') %}
```

Comment:
Why "undefined" instead of "This field is empty" ? IMHO we should be consistent with other parts of the system.

Reply:
Well it was on design like this but it changes, so for now we dont have undefined.